packages/inspector/src/internal-inspect.ts

```
@@ -0,0 +1,206 @@
+
+    // last chance to check if it's an object
+    if (value === Object(value)) {
+        return inspectObject(value as object, options, inspect, indent, depth);
+    }
+
+    // We have run out of options! Just stringify the value
+    return (options as Options).stylize(String(value), type);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
+export const registerConstructor = (constructor: Function, inspector: Inspect): boolean => {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)